### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by kant**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,21 +6,21 @@
...
 * You are not satisfied with the accuracy of the OCR, and want to ask how you can improve it. Note: You should first read the [ImproveQuality](https://github.com/tesseract-ocr/tesseract/wiki/ImproveQuality) wiki page. 
-* You are trying to train Tesseract and you have a problem and/or want to ask a question about the training process.  Note: You should first read the **official** guides [[1]](https://github.com/tesseract-ocr/tesseract/wiki/tesstrain.sh) or [[2]](https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract) found in the project wiki.
+* You are trying to train Tesseract and you have a problem and/or want to ask a question about the training process.  Note: You should first read the **official** guides [[1]](https://github.com/tesseract-ocr/tesseract/wiki) or [[2]](https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract) found in the project wiki.
 * You have a general question. 
 
 An issue should only be reported if the platform you are using is one of these:
   * Linux (but not a version that is more than 4 years old)
   * Windows (Windows 7 or newer version)
-  * Mac (last 3 releases)
+  * Mac OSX (last 3 releases)
```
Comment (stweil) [inline CONTRIBUTING.md]: Isn't the official name `macOS`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,21 +6,21 @@
...
 
 For older versions or other operating systems, use the Tesseract forum.
 
-When creating an issue, please report your operating system, including its specific version: "Ubuntu 16.04", "Windows 10", "OS X 10.11" etc.
+When creating an issue, please report your operating system, including its specific version: "Ubuntu 16.04", "Windows 10", "Mac OSX 10.11" etc.
```
Comment (stweil) [inline CONTRIBUTING.md]: `macOS`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -6,21 +6,21 @@
...
+Use the [users mailing-list](https://groups.google.com/forum/#!forum/tesseract-ocr) instead of creating an Issue if ...
 * You have problems using Tesseract and need some help.
 * You have problems installing the software.
 * You are not satisfied with the accuracy of the OCR, and want to ask how you can improve it. Note: You should first read the [ImproveQuality](https://github.com/tesseract-ocr/tesseract/wiki/ImproveQuality) wiki page. 
-* You are trying to train Tesseract and you have a problem and/or want to ask a question about the training process.  Note: You should first read the **official** guides [[1]](https://github.com/tesseract-ocr/tesseract/wiki/tesstrain.sh) or [[2]](https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract) found in the project wiki.
+* You are trying to train Tesseract and you have a problem and/or want to ask a question about the training process.  Note: You should first read the **official** guides [[1]](https://github.com/tesseract-ocr/tesseract/wiki) or [[2]](https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract) found in the project wiki.
```
Comment (stweil) [inline CONTRIBUTING.md]: I‌ think the first link should be https://github.com/tesseract-ocr/tesseract/wiki/Training-Tesseract-%E2%80%93-tesstrain.sh. It was renamed with Wiki commit a8d1ec6bcc941c3564666a6b12ab7208285cacbd.

**Comment by amitdo** [comment]
@kant, can you reply to @stweil comments?

**Comment by rfschtkt** [comment]
[Wikipedia](https://en.wikipedia.org/wiki/MacOS) indicates "Mac OS X" -> "OS X" -> "macOS", not "OS X" -> "Mac OSX".

**Comment by amitdo** [comment]
@stweil,

Since the OP is not responding, I suggest you to make a new PR.

**Pull request closed without merging**